src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -14,4 +16,10 @@
 
     Optional<List<User>> findByBirthdateBetweenAndNameIgnoreCaseContaining(LocalDate start, LocalDate end,
             String string);
+
+    @Query("SELECT u FROM USERS u WHERE (:dateFrom IS NULL OR u.birthdate >= :dateFrom) AND (:dateTo IS NULL"
+            + " OR u.birthdate <= :dateTo) AND (:someChars IS NULL OR LOWER(u.name) LIKE CONCAT('%',LOWER(:someChars),'%'))")
+    Optional<List<User>> findUserByBirthdayBetweenAndContaining(@Param("dateFrom") LocalDate dateFrom,
```

Comment:
Same here, Optional shouldn't be used with a List

Reply:
Done :D